lib/linguist/languages.yml

```diff
@@ -6842,6 +6842,17 @@ SuperCollider:
   tm_scope: source.supercollider
   ace_mode: text
   language_id: 361
+SurrealQL:
+  type: programming
+  color: "#ff00a0"
+  aliases:
+  - surql
+  extensions:
+  - ".surql"
+  - ".surrealql"
```

Comment:
This is no where near popular enough for inclusion as there are only 4 file indexed by the search engine.

```suggestion
```

Reply:
That's alright, we can exclude that file extension if you'd like